.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue requests
+a new topic, preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics and the creation of new topics.
+
+This way, we have those discussions before significant
+effort is spent writing a new topic. Click on the Guidelines
+for Contributing link above for details.
+
+## Issues with Existing Topics
+
+If the issue is a simple typo or similar correction, just submit a PR.
+Otherwise, let us know what's wrong or what we should improve. Include a
+link to the topic.
+
+## Requests for new Topics
+
+Tell us where this topic should go in the Table of Contents. Consider where you
```

Comment:
should this be one of the steps? I don't think it should be the first thing when requesting a new topic.